### PR TITLE
fix: type error

### PR DESCRIPTION
**Pull request by haxxmaxx**
Also added a test for null

### CODE REVIEW

File: src/table/utils/color-utils.ts [hunk 1]
```diff
@@ -46,8 +46,8 @@ export function resolveToRGBAorRGB(input: string): string {
   return 'none';
 }
 
-export function isDarkColor(color: string | undefined): boolean {
-  if (color === undefined) return false;
+export function isDarkColor(color: string | undefined | null): boolean {
+  if (color == null) return false;
```
Comment (LiKang6688) [inline src/table/utils/color-utils.ts:50]: `undefined == null` is true, not sure if there is any side effect.

**Review by LiKang6688 — APPROVED** [review]

File: src/table/utils/color-utils.ts (hunk 1 above)
Comment (haxxmaxx) [reply src/table/utils/color-utils.ts:50]: that is the intention, so it catches both undefined and null

Comment (LiKang6688) [reply src/table/utils/color-utils.ts:50]: Yes, i know. (color === undefined && color === null) is more common. But it should be fine
